bfcfg

```diff
@@ -59,6 +59,12 @@ mfg_cfg()
     return
   fi
 
+  if [ -n "${MFG_OOB_MAC}" -a -z "${MFG_OPN_STR}" ] ||
+     [ -z "${MFG_OOB_MAC}" -a -n "${MFG_OPN_STR}" ]; then
+    echo "ERROR: both MFG_OOB_MAC and MFG_OPN_STR must be specified"
+    echo "       in order to write information to the MFG partition."
```

Comment:
Maybe we could use "log_msg" instead of echo to be consistent?
When pushing bf.cfg together with BFB, the 'echo' command is not showing anywhere and this error will be lost.